tests/acceptance/01_vars/01_basic/os_version_minor_simple.cf

```diff
@@ -0,0 +1,40 @@
+body common control
+{
+  bundlesequence => { "check" };
+}
+
+bundle agent check
```

Comment:
This test is a ticking timebomb that will stop working once these platforms come out with a new releases. Can you come up with a more dynamic way to test this?